doppler/assets/oauth_clients.json

```
@@ -0,0 +1,13 @@
+{
+  "integration": {
+    "scopes": ["api_keys_write"],
+    "client_role": "integration",
+    "name": "Doppler API",
+    "onboarding_url": "https://dashboard.doppler.com/notification_services/datadog/connect",
+    "description": "Doppler API Client",
+    "redirect_uris": [
+      "https://dashboard.doppler.com/notification_services/datadog/callback"
+    ],
+    "id": "a9a59c10-d841-4c8e-8615-202bd9ea3599"
```

```suggestion
    "id": "87fdb7f6-542c-11ef-abb5-da7ad0900002"
```
This is the correct uuid.

---

@london-wharton, @emarsha94: I'm having some trouble confirming that either of those UUIDs belongs to our production Datadog app. Can you confirm if it's associated with this app? https://app.datadoghq.com/apps/83469c40-caa8-11ee-8925-da7ad0900002/edit/oauth/8d0c113c-caa9-11ee-8b55-da7ad0900002

---

Correct - the published client id for that app is 87fdb7f6-542c-11ef-abb5-da7ad0900002

---

Brilliant, thanks for the assist!